### PR TITLE
Kuba/5293 sync zoom by wheel and zoom by right-click

### PR DESCRIPTION
**Pull request by jszalek**

### CODE REVIEW

File: beakerx/js/src/plot/plotScope.js [hunk 1]
```diff
@@ -1626,7 +1650,8 @@ define([
     var d3trans = d3.event.transform || d3.event;
     self.lastx = d3trans.x;
     self.lasty = d3trans.y;
-    var newK = ((d3trans.k-1)*0.5)+1;
+    // var newK = ((d3trans.k-1)*0.5)+1;
```
Comment (scottdraves) [inline beakerx/js/src/plot/plotScope.js]: do not leave in commented out code

File: beakerx/js/src/plot/plotScope.js [hunk 2]
```diff
@@ -2181,7 +2211,8 @@ define([
     };
 
     self.resetSvg();
-    self.zoomObj = d3.zoom();
+    self.zoomObj = d3.zoom().scaleExtent([0.7, Infinity]);
+    // self.zoomObj = d3.zoom();
```
Comment (scottdraves) [inline beakerx/js/src/plot/plotScope.js]: delete do not comment out

File: beakerx/js/src/plot/plotScope.js (hunk 1 above)
Comment (jszalek) [reply beakerx/js/src/plot/plotScope.js]: removed

File: beakerx/js/src/plot/plotScope.js (hunk 2 above)
Comment (jszalek) [reply beakerx/js/src/plot/plotScope.js]: removed

**Pull request merged**